ai_traineree/agents/dummy.py

```diff
@@ -0,0 +1,178 @@
+import copy
+import numpy as np
+from ai_traineree.agents import AgentBase
+from ai_traineree.types import AgentState, BufferState, DataSpace, NetworkState
+from typing import Callable, Dict, Optional, Type
+from ai_traineree.loggers import DataLogger
+from ai_traineree.experience import Experience
+
+import warnings
+
+class DummyAgent(AgentBase):
+    """Deep Q-Learning Network (DQN).
+     
+     Agent that returns random values regardless of the input
+     Print mean of the taken actions, observables and reward
+     
+     """
```

Comment:
This likely needs an update. Maybe something like

```
"""Dummy Agent.

Agent that returns random values in specified shapes.
"""
````

Reply:
Agree! On it